recipes/remove-dd-agent.rb

```diff
@@ -1,13 +1,52 @@
+# Copyright:: 2011-Present, Datadog
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
 # Run this recipe to completely remove the Datadog Agent
 
 # On Windows, this recipe works only with Chef >= 12.6
 
+# Importing apt_sources_list_file variable from recipe_helpers.rb
+apt_sources_list_file = Chef::Datadog.apt_sources_list_file
+
 case node['os']
 when 'linux'
+  # First remove the datadog-agent package
   package 'datadog-agent' do
     action :purge
   end
+
+  # Then remove the installation files (depending on the OS: sources_list file, datadog-signing-keys package...)
+  case node['platform_family']
+  when 'amazon', 'rhel' # 'rhel' includes redhat, centos, rocky, scientific and almalinux
```

Comment:
I think you're missing `fedora` here, this should match https://github.com/DataDog/chef-datadog/blob/main/recipes/repository.rb#L165

Reply:
Forgot to put it back, mb ! Fixed in [5c87ee8](https://github.com/DataDog/chef-datadog/pull/865/commits/5c87ee892010d8c3d9c05c6e3bc86113cd1412e7).